### PR TITLE
cleanup coins.c

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: src/coins.c [hunk 1]
```diff
@@ -1,14 +1,15 @@
+#include "global.h"
```
Comment (tgsm) [inline src/coins.c:1]: This is already included by the compiler

Comment (red031000) [reply src/coins.c:1]: sure it is, but it causes errors in the IDE if I don't include it

Comment (adrienntindall) [reply src/coins.c:1]: Then get a better IDE or push without the change smh

Comment (red031000) [reply src/coins.c:1]: genuinely, all c files should include global.h imo

Comment (red031000) [reply src/coins.c:1]: (exception to this being libs ofc)

Comment (adrienntindall) [reply src/coins.c:1]: ok, remove it here so we can merge pls

Comment (red031000) [reply src/coins.c:1]: this isn't a lib tho??? stuff in global.h is referenced in the file?

Comment (adrienntindall) [reply src/coins.c:1]: sry was in response to your second comment, we're not gonna include global.h in everything

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**